packages/sdk/client-protocol/src/echo.ts

```diff
@@ -45,7 +47,7 @@ export interface Echo extends MulticastObservable<Space[]> {
   /**
    * Creates a new space.
    */
-  create(): Promise<Space>;
+  create(meta?: PropertiesProps): Promise<Space>;
```

Comment:
PropertiesProps or SpaceProps?

Reply:
I believe they're PropertiesProps because they're used as initial values for the space properties